### PR TITLE
added landing page UI changes

### PR DESCRIPTION
**Pull request by moki298**
#283 changes added

### CODE REVIEW

**Comment by angle943** [comment]
hey @moki298 , sorry about not responding to the issue.

Can we update the Nav Bar so that it doesn't have a max-width? The `TheNewBoston` should always be on the top left, and the menus on the top-right should always be right-adjusted

File: src/containers/Home/HomeHero/index.tsx [hunk 1]
```diff
@@ -3,24 +3,28 @@ import React, {FC} from 'react';
...
     <div className="HomeHero">
-      <svg className="HomeHero__background-graphic" viewBox="0 0 1366 396">
-        <path fill="#F6F9FC" fillRule="nonzero" d="M0 395.5L1366 106V0H0v395.5z" />
-      </svg>
       <div className="HomeHero__wrapper">
         <div className="HomeHero__left">
           <div className="HomeHero__left-content-container">
             <h1 className="HomeHero__title">
-              <span>Build software.</span>
+              <span>
```
Comment (angle943) [inline src/containers/Home/HomeHero/index.tsx:18]: do we need this `span` inside the `h1`?

File: src/containers/Home/HomeHero/index.tsx [hunk 2]
```diff
@@ -3,24 +3,28 @@ import React, {FC} from 'react';
...
+                <b>Build</b> software.
```
Comment (angle943) [inline src/containers/Home/HomeHero/index.tsx:19]: lets use `<strong>` over `<b>`

File: src/containers/Home/HomeHero/index.tsx [hunk 3]
```diff
@@ -3,24 +3,28 @@ import React, {FC} from 'react';
...
+              </span>
               <br />
-              <span>Earn digital currency.</span>
+              <span>
```
Comment (angle943) [inline src/containers/Home/HomeHero/index.tsx:22]: do we need this `span`?

File: src/containers/Home/HomeInstantTransactions/index.tsx [hunk 4]
```diff
@@ -12,36 +12,38 @@ const HomeInstantTransactions: FC = () => {
...
-              We rebuilt the Blockchain from the ground up. We kissed those outdated PoW and PoS algorithms goodbye and
-              replaced them with an innovative trust based architecture.
-            </p>
-            <NavLink className="HomeInstantTransactions__content-container--display-large" to="/guide/introduction">
-              <Button>View Docs</Button>
-            </NavLink>
+        <div className="HomeInstantTransactions__summary-container">
+          <div className="HomeInstantTransactions__summary-snippet-container">
+            <h1 className="HomeInstantTransactions__title">
+              <b>Say goodbye to 10 minute block times</b>
```
Comment (angle943) [inline src/containers/Home/HomeInstantTransactions/index.tsx:18]: this <b> tag can be applied to the styling for `HomeInstantTransactions__title`

File: src/containers/Home/HomeInstantTransactions/index.tsx [hunk 5]
```diff
@@ -12,36 +12,38 @@ const HomeInstantTransactions: FC = () => {
...
+            </h1>
+            <div className="HomeInstantTransactions__text-snippets">
+              <div>
```
Comment (angle943) [inline src/containers/Home/HomeInstantTransactions/index.tsx:21]: extra div

**Comment by moki298** [comment]
> hey @moki298 , sorry about not responding to the issue.
> 
> Can we update the Nav Bar so that it doesn't have a max-width? The `TheNewBoston` should always be on the top left, and the menus on the top-right should always be right-adjusted

Hey @angle943 this is contrary to what is being said in the last check list point at #283 , can you please look at it once and confirm? Thanks.

**Comment by angle943** [comment]
yeah sorry @moki298 , ignore what I said about the Nav bar. @kristykjlee made a comment on the issue, you can follow that

**Comment by moki298** [comment]
@angle943 all the required changes are updated/added.

**Comment by angle943** [comment]
Screen Shot 2020-10-26 at 9 12 21 PM

hmm is your page looking like this?

**Comment by moki298** [comment]
This is how it looks on my local, not sure which space you are talking about.

Screen Shot 2020-10-27 at 12 15 57 AM

**Comment by moki298** [comment]
Ok, I can see what you are talking about. let me do a quick fix here.

**Comment by moki298** [comment]
@angle943 this is ready to be merged now, #325 has to be removed if this has to be merged, both these tasks involve adding a footer, which lead to these conflicts. I can fix these conflicts but by just fixing these conflicts and merging this into master would add duplicate footer components and other code duplication(unwanted complexity). The best way to merge would be to remove #325 changes from master and then merge this and then add #325 changes on top of master as required(might need  little tweaks). If there is a better way please let me know I am more than happy to help. Thanks!

**Comment by moki298** [comment]
@angle943 Ok, I can try and add #325  changes on this branch(other way to resolve these conflicts). But before that can you confirm if you want me to do that? Or do you have any other plans?

**Pull request merged**